doc/release-notes/5.11.1-release-notes.md

```diff
@@ -0,0 +1,72 @@
+# Dataverse Software 5.11.1
+
+This is a bug fix release of the Dataverse Software. Dataverse Software v5.11 will no longer be made available and installations should upgrade directly from v5.10.1 to v5.11.1.
+
+## Release Highlights
+
+Dataverse Software 5.11 contains two critical issues that are fixed in this release.
+
+First, if you delete a file from a published version of a dataset that has restricted files, the file will be deleted from the file system (or S3) and lose its "owner id" in the database. For details, see Issue #8867.
+
+Second, if you are a superuser, it's possible to click "Delete Draft" and delete a published dataset if it has restricted files. For details, see #8845 and #8742.
+
+## Notes for Dataverse Installation Administrators
+
+### Identifying Datasets with Deleted Files
+
+If you have been running 5.11, check if any files show "null" for the owner id. The "owner" of a file is the parent dataset:
+
+```
+select * from dvobject where dtype = 'DataFile' and owner_id is null;
+```
+
+For any of these files, change the owner id to the database id of the parent dataset. In addition, the file on disk (or in S3) is likely gone. Look at the "storageidentifier" field from the query above to determine the location of the file then restore the file from backup.
+
+### Identifying Datasets Superusers May Have Accidentally Destroyed 
+
+Check the "actionlogrecord" table for DestroyDatasetCommand. While these "destroy" entries are normal when a superuser uses the API to destroy datasets, an entry is also created if a superuser has accidentally deleted a published dataset in the web interface with the "Delete Draft" button.
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.11.1 Milestone](https://github.com/IQSS/dataverse/milestone/105?closed=1) in GitHub.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Community Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](https://guides.dataverse.org/en/5.11.1/installation/). Please also contact us to get added to the [Dataverse Project Map](https://guides.dataverse.org/en/5.11.1/installation/config.html#putting-your-dataverse-installation-on-the-map-at-dataverse-org) if you have not done so already.
+
+## Upgrade Instructions
+
+0\. These instructions assume that you've already successfully upgraded from Dataverse Software 4.x to Dataverse Software 5 following the instructions in the [Dataverse Software 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0). After upgrading from the 4.x series to 5.0, you should progress through the other 5.x releases before attempting the upgrade to 5.11.1.
+
```

Comment:
To upgrade from 5.10.1, follow the instructions for installing release 5.11 using the v5.11.1 war file. If you had previously installed v5.11 (no longer available), follow the simplified instructions below.

Reply:
This looks great, I'll go ahead and add this. I will also add this sentence/some form thereof to the very top of the release note.